cmake/configure/configure_boost.cmake

```diff
@@ -47,6 +47,19 @@ MACRO(FEATURE_BOOST_CONFIGURE_BUNDLED)
     LIST(APPEND BOOST_DEFINITIONS "BOOST_ALL_NO_LIB")
     LIST(APPEND BOOST_USER_DEFINITIONS "BOOST_ALL_NO_LIB")
   ENDIF()
+
+  #
+  # Our bundled boost version 1.62 checks for the availability of
+  # "emplace_hint" incorrectly. Thus, simply define
```

Comment:
-> incorrectly: It tests for the preprocessor define BOOST_NO_CXX11_HDR_UNORDERED_MAP in bundled/boost-.../include/boost/serialization/map.h thinking that that define is characteristic for the presence of std::(multi)map::emplace_hint. This is generally correct, except for GCC 4.8, for which the preprocessor variable is defined, but the function does not exist. (See also https://svn.boost.org/trac/boost/ticket/12755 .)